src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images\logo.png"
+        alt="logotype Moyo"
+        class="logo__image">
```

Comment:
You need to check and fix it everywhere
<img width="1079" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/7e4ca243-a843-4c89-a183-913d53b8ee3d">